pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -47,71 +47,99 @@ type KubeletConfiguration struct {
 
 	// staticPodPath is the path to the directory containing local (static) pods to
 	// run, or the path to a single static pod file.
+	// Dynamic Kubelet Config: If dynamically updating this field, consider that
+	// the set of static pods specified at the new path may be different than the
+	// ones the Kubelet initially started with, and this may disrupt your node.
```

Comment:
This documentation seems to be pointing at a different problem - most things don’t work with dynamic config.  I’m not sure documenting the problem to our users is the right approach.

Reply:
I really have trouble envisioning supporting a user trying to leverage this feature.  This seems like a giant loaded explosive ready to go off under the feet of a user making use of this feature.  For 90% of these, if I have to restart the kubelet anyway or manage the lifecycle of the kubelet... why do I need dynamic config?